homu/init.sls

```diff
@@ -46,3 +46,11 @@ homu:
     - require:
       - pip: homu
       - file: /home/servo/homu/cfg.toml
+
+/var/homu:
+  file.directory:
+    - source:  salt://{{ tpldir }}/var/homu
```

Comment:
We just want the directory to exist, so no need for a `source` since we're not filling it with anything.